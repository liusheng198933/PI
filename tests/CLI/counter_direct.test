#!/bin/sh
./run_one_test.py $srcdir/testdata counter_direct stats.json
