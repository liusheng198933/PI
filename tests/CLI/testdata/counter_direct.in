counter_write ExactOne_counter NEXT_ENTRY bytes=388
table_add ExactOne 0 => actionA 0
counter_read ExactOne_counter 0
counter_reset ExactOne_counter 0
counter_read ExactOne_counter 0
counter_write ExactOne_counter 0 packets=7 bytes=9993
counter_read ExactOne_counter 0
counter_write ExactOne_counter 0 packets=12
counter_read ExactOne_counter 0
counter_reset ExactOne_counter 0
counter_read ExactOne_counter 0
