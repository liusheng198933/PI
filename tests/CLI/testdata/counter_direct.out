????
Device assigned successfully.
Selecting device.
????
Dumping counter entry:
	Bytes: 388
Direct resource spec was stored.
????
Entry was successfully added with handle 0.
????
Dumping counter entry:
	Packets: 0
	Bytes: 388
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 7
	Bytes: 9993
????
Dumping counter entry:
	Packets: 12
????
Dumping counter entry:
	Packets: 12
	Bytes: 9993
????
????
Dumping counter entry:
	Packets: 0
	Bytes: 0
????
